css/style.css

```diff
@@ -550,17 +550,20 @@ figcaption details p {
 }
 
 .floe .floe-image {
-    width: 325px;
-    height: 325px;
+    width: 270px;
+    display: block;
+    height: 280px;
     margin: 0 auto;
     margin-bottom: 2em;
+    
+
 }
 
-.floe .floe-image-UIO             { background: url("../images/floe-02.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-exploreTool     { background: url("../images/floe-01.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-cloud           { background: url("../images/floe-05.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-preferenceIcons { background: url("../images/floe-04.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-videoPlayer     { background: url("../images/floe-03.png") no-repeat 0 0 / 325px 325px; }
+.floe .floe-image-UIO             { background: url("../images/floe-02.png") no-repeat 0 0 / 270px 280px; }
+.floe .floe-image-exploreTool     { background: url("../images/floe-01.png") no-repeat 0 0 / 270px 280px; }
+.floe .floe-image-cloud           { background: url("../images/floe-05.png") no-repeat 0 0 / 270px 280px; }
+.floe .floe-image-preferenceIcons { background: url("../images/floe-04.png") no-repeat 0 0 / 270px 280px; }
+.floe .floe-image-videoPlayer     { background: url("../images/floe-03.png") no-repeat 0 0 / 270px 280px; }
```

Comment:
Instead of changing the sizes of images explicitly, you should consider using max-width property and defining those sizes in percentages. This way the image scales itself as per the size and doesn't abruptly change its size on reaching a specific screen size. Look at #128. This issue has been extensively discussed in #119 and a few other places.